pkg/cli/args.go

```diff
@@ -0,0 +1,67 @@
+package cli
+
+import (
+	libflag "flag"
+	"io"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+)
+
+const (
+	OneDashFlag    NormalizeActsType = iota
+	DoubleDashFlag NormalizeActsType = iota
+)
+
+type NormalizeActsType byte
+
+type Args struct {
+	cli.Args
+}
+
+func newArgs(args []string) *Args {
+	flagSet := libflag.NewFlagSet("", libflag.ContinueOnError)
+	flagSet.SetOutput(io.Discard)
+	flagSet.Parse(append([]string{"--"}, args...))
+
+	return &Args{
+		Args: cli.NewContext(nil, flagSet, nil).Args(),
+	}
+
+}
+
+// Normalize formats the arguments according to the given actions.
+func (args *Args) Normalize(acts ...NormalizeActsType) *Args {
+	var strArgs []string
+
+	for _, arg := range args.Slice() {
+		arg := arg
```

Comment:
Why do we need this? I don't see any goroutine or other closure that requires us to rescope this?

Reply:
Ah, I just forgot to remove, after modifying the code. Thanks!